packages/terra-progress-bar/docs/README.md

```diff
@@ -20,6 +20,7 @@ import ProgressBar from 'terra-progress-bar';
   heightSize="tiny"
   value={3}
   max={10}
+  color="Orange"
```

Comment:
nit: change this to a hexidecimal value, e.g. `#06dce8` rather than a color keyword.